examples/cluster-configuration/services-configuration.yaml

```diff
@@ -22,6 +22,9 @@
 #    cluster-id: pai
 #    cluster-type: yarn
 #
+#    # Gateway address, used to visit the cluster from public network
+#    gateway-uri: "https://pai.domain.com"
```

Comment:
what‘s difference with pylon.domain

Reply:
pylon may not have a public ip or domain
```
user <-> gateway <-> pylon
```